crates/llms/src/embeddings/mod.rs

```diff
@@ -44,6 +44,9 @@ pub enum Error {
         source: Box<dyn std::error::Error + Send + Sync>,
     },
 
+    #[snafu(display("Invalid value for `pooling` parameter: {value}.\nSupported pooling value include: `cls`, `mean`, `splade`, `last_token`. Verify `pooling` parameter and try again."))]
```

Comment:
```suggestion
    #[snafu(display("Invalid value for `pooling` parameter: {value}.\nSpecify a valid pooling value of `cls`, `mean`, `splade`, or `last_token`."))]
```

We can combine the last 2 sentences into a single action. `Allowed values are: {}. Check it`, vs `Check the value is: {}`

Reply:
Is there a details page we can also link to in docs?

Reply:
There aren't great docs. Mainly https://www.sbert.net/docs/package_reference/sentence_transformer/models.html#sentence_transformers.models.Pooling